tox.ini

```diff
@@ -29,6 +29,15 @@ deps =
     pytest-mock
     pytest-html
 
+[testenv:py27]
+deps =
+    pytest
+    pytest-cov
+    pytest-remove-stale-bytecode
+    pytest-mock
+    pytest-html
```

Comment:
To be DRY you could replace all dependencies besides `DateTime` by `{[testenv]deps}`.